centos/SPECS/softethervpn.spec

```diff
@@ -1,7 +1,7 @@
 %define majorversion 4
-%define minorversion 19
-%define buildversion 9582
-%define dateversion 2015.10.06
+%define minorversion 23
+%define buildversion 9647
```

Comment:
9651 has already been released

Reply:
https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/ChangeLog
it is said 9647:(